test/sass/_color.scss

```diff
@@ -45,4 +45,20 @@
     @include assert-equal($test, $expect,
       'Lightens a dark color');
   }
+
+  @include test('Get Color (Blue) [function]') {
+    $test: get-color('blue');
+    $expect: #1779ba;
```

Comment:
We do not expect _always `#1779ba`_, but only _the color for blue from the color map_. So please set `$foundation-color: (blue: #1779ba);`. Same for `get-state`.